test/adapter/json/collection_test.rb

```diff
@@ -84,6 +84,72 @@ def test_include_option
 
           assert_equal(expected, actual)
         end
+
+        def test_fields_option
+          serializer = ActiveModel::Serializer::CollectionSerializer.new([@first_post, @second_post])
+          adapter = ActiveModelSerializers::Adapter::Json.new(serializer)
+          actual = adapter.serializable_hash(fields: [:id])
```

Comment:
these options should be passed into the adapter.  most conventional way to do it is

````ruby
actual = serializable([@first_post, @second_post], adapter: :json, fields: [:id]).serializable_hash
```

see https://github.com/rails-api/active_model_serializers/issues/1643